pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/ReportManager.kt

```diff
@@ -239,7 +238,25 @@ class ReportManager: KoinComponent {
             } else
                 this.content = content
         }
-       return  createReportItem(uploadId,deadLetterReportId,deadLetterReport)
+        return  createReportItem(uploadId,deadLetterReportId,deadLetterReport)
+    }
+
+    /**
+     * Creates a dead-letter report if there is a malformed JSON. This is called if DISABLE_VALIDATION is true
+     *
+     * @param deadLetterReason String
+     * @return String
+     * @throws BadStateException
+     */
+
+    @Throws(BadStateException::class)
+    fun createDeadLetterReport(deadLetterReason: String): String {
+        val deadLetterReportId = UUID.randomUUID().toString()
+        val deadLetterReport = ReportDeadLetter().apply {
+            this.id = deadLetterReportId
+            this.deadLetterReason = deadLetterReason
```

Comment:
For this use this.deadLetterReasons = listOf(deadLetterReason) and get rid of the deadLetterReason.